### ISSUE TITLE
Adroid build issue since 4 November 2022

### ISSUE THREAD

**Issue by keremoge** [jitsi/jitsi-meet#12520]
### Description:
There is a build issue since 4 November, as it is stated here:
<img width="1262" alt="image" src="https://user-images.githubusercontent.com/4692074/200255140-2febb858-014b-4efb-8931-7dc679a07836.png">

### Steps to reproduce:

1. try to build Android project

### Expected behavior:

to be built

### Actual behavior:

 build error

**Comment by keremoge** [comment]
I will do a PR about this issue

**Comment by saghul** [comment]
What's the build issue? We haven't run into any.

**Comment by saghul** [comment]
For reference: https://github.com/facebook/react-native/issues/35210

**Comment by keremoge** [comment]
I checked it, there is no build error for jitsi-meet android on RN0.68.4. 
So this issue is not affected jitsi-meet.

Early this morning I got error from my own project when I saw https://github.com/facebook/react-native/issues/35210 I upgraded RN 0.68.5 and fixed the issue so I thought that it is a problem for jitsi-meet too.

So there is no urgent need to update R to 0.68.4, but still I will make a PR : )

I will close this issue
